frontend/cypress/integration/common/kiali_alert.ts

```diff
@@ -0,0 +1,16 @@
+import { Then } from '@badeball/cypress-cucumber-preprocessor';
+
+Then(`user should see no Istio Components Status`, () => {
+  cy.intercept({
+    pathname: '**/api/istio/status*',
+    query: {
+      objects: ''
+    }
+  }).as('istioStatus');
+
+  cy.request('GET', '/api/istio/status')
```

Comment:
IMO this request will not happen in Kiali UI side, but from cypress itself, so the next wait method will not be satisfied.

Reply:
you might need to click refresh button here instead of cy.request

Reply:
Yep. I checked the Cypress docs and you are right. The cy.request is now replaced.